python/dask_cudf/dask_cudf/core.py

```diff
@@ -30,6 +31,8 @@
     DatetimeAccessor,
 )
 
+DASK_VERSION = LooseVersion(dask.__version__)
```

Comment:
We're super dependent on newer versions of dask already, so I'd be perfectly happy just bumping our dependencies in the conda recipe / environment yml files and calling this a day instead of trying to be backwards compatible.

Reply:
Makes sense, but the related PR in dask hasn't been merged yet, let alone released.

Reply:
Understood, we can merge this as is and remove at a later point.